### ISSUE TITLE
Depreciation warning on Ember 2.1.0

### ISSUE THREAD

**Issue by jniechcial** [jerel/ember-cli-growl#8]
I bumped into this depreciation after updating Ember to 2.1.0:

```
DEPRECATION: The `initialize` method for Application initializer 'growl' should take only one argument - `App`, an instance of an `Application`.
```

The easiest fix would be to just change initializer. However, I think that it is rather more useful to remove initializer at all and provide just a service object. Thus, user would inject it where needed not in each route and controller.

What do you think?

**Comment by jerel** [comment]
I agree. If you want to make a pull request I'm happy to review it and tag a new version

**Comment by jniechcial** [comment]
@jerel Sorry for delay but I couldn't find any time to focus on ember-cli-growl. I am working on that but I still haven't finished and from my plans for this week I see that I will start again at most on next Monday. I also bumped into another issue while upgrading to ember-cli 1.13.12 regarding css import in `index.js`. I will try to cover it also. Please give me a sign if you would work on these cases.
